sorc/build_gsi_enkf.sh

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env bash
+set -eux
+
+cwd=$(pwd)
+
+OPTIND=1
+while getopts ":dov" option; do
+  case "${option}" in
+    d) export BUILD_TYPE="DEBUG";;
+    o) _ops="YES";;
+    v) export BUILD_VERBOSE="YES";;
+    \?)
+      echo "[$BASH_SOURCE]: Unrecognized option: ${option}"
+      usage
+      ;;
+    :)
+      echo "[$BASH_SOURCE]: ${option} requires an argument"
+      usage
+      ;;
+  esac
+done
+shift $((OPTIND-1))
+
+#TODO - remove UTIL_OPTS line after GSI removes all utilities from its repo.
+BUILD_TYPE=${BUILD_TYPE:-"Release"} \
+BUILD_VERBOSE=${BUILD_VERBOSE:-"YES"} \
```

Comment:
Default should be NO so the -v option actually does something, but I think having it at all is unnecessary since the default if not set is no anyway.